content/en/docs/setup/independent/high-availability.md

```diff
@@ -1,536 +1,502 @@
+              advertise-client-urls: "https://CP1_IP:2379"
+              listen-peer-urls: "https://CP1_IP:2380"
+              initial-advertise-peer-urls: "https://CP1_IP:2380"
+              initial-cluster: "CP0_HOSTNAME=https://CP0_IP:2380,CP1_HOSTNAME=https://CP1_IP:2380"
+              initial-cluster-state: existing
+            serverCertSANs:
+              - CP1_HOSTNAME
+              - CP1_IP
+            peerCertSANs:
+              - CP1_HOSTNAME
+              - CP1_IP
+        networking:
+            # This CIDR is a calico default. Substitute or remove for your CNI provider.
+            podSubnet: "192.168.0.0/16"
+
+1. Replace the following variables in the template that was just created with
+   values for your specific situation:
+
+    - `LOAD_BALANCER_DNS`
+    - `LOAD_BALANCER_PORT`
+    - `CP0_HOSTNAME`
+    - `CP0_IP`
+    - `CP1_HOSTNAME`
+    - `CP1_IP`
+
+1. Move the copied certificates to the proper locations
+
+  ```sh
+  USER=ubuntu # customizable
+  mkdir -p /etc/kubernetes/pki/etcd
+  mv /home/${USER}/ca.crt /etc/kubernetes/pki/
+  mv /home/${USER}/ca.key /etc/kubernetes/pki/
+  mv /home/${USER}/sa.pub /etc/kubernetes/pki/
+  mv /home/${USER}/sa.key /etc/kubernetes/pki/
+  mv /home/${USER}/front-proxy-ca.crt /etc/kubernetes/pki/
+  mv /home/${USER}/front-proxy-ca.key /etc/kubernetes/pki/
+  mv /home/${USER}/etcd-ca.crt /etc/kubernetes/pki/etcd/ca.crt
+  mv /home/${USER}/etcd-ca.key /etc/kubernetes/pki/etcd/ca.key
+  mv /home/${USER}/admin.conf /etc/kubernetes/admin.conf
+  ```
+
+1. Run the kubeadm phase commands to bootstrap the kubelet
+
+  ```sh
+  kubeadm alpha phase certs all --config kubeadm-config.yaml
+  kubeadm alpha phase kubelet config write-to-disk --config kubeadm-config.yaml
+  kubeadm alpha phase kubelet write-env-file --config kubeadm-config.yaml
+  kubeadm alpha phase kubeconfig kubelet --config kubeadm-config.yaml
+  systemctl start kubelet
+  ```
+
+1. Run the commands to add the node to the etcd cluster
+
+  {{< note >}}
+  **Note**: This step will cause the etcd cluster to become unavailable for a
+  brief period between adding the node to the running cluster config and the new
+  node joining the etcd cluster.
+  {{< /note >}}
+
+  ```sh
+  CP0_IP=10.0.0.7
+  CP0_HOSTNAME=cp0
+  CP1_IP=10.0.0.8
+  CP1_HOSTNAME=cp1
+
+  KUBECONFIG=/etc/kubernetes/admin.conf kubectl exec -n kube-system etcd-${CP0_HOSTNAME} -- etcdctl --ca-file /etc/kubernetes/pki/etcd/ca.crt --cert-file /etc/kubernetes/pki/etcd/peer.crt --key-file /etc/kubernetes/pki/etcd/peer.key --endpoints=https://${CP0_IP}:2379 member add ${CP1_HOSTNAME} https://${CP1_IP}:2380
+  kubeadm alpha phase etcd local --config kubeadm-config.yaml
+  ```
+
+1. Deploy the control plane components and mark the node as a master
+
+  ```sh
+  kubeadm alpha phase kubeconfig all --config kubeadm-config.yaml
+  kubeadm alpha phase controlplane all --config kubeadm-config.yaml
+  kubeadm alpha phase mark-master --config kubeadm-config.yaml
```

Comment:
@Bradamant3 Technically, since we are requiring a load balancer for this and overriding the controlPlaneEndpoint these steps can be moved up above the etcd add step. I'm not sure if it makes sense to go ahead and consolidate these above or to leave them separate to make it more easily adaptable for the non-lb (or lb configured after install) approach...